src/plugins/intel_npu/src/plugin/npuw/llm_compiled_model.cpp

```diff
@@ -610,13 +701,26 @@ ov::npuw::LLMCompiledModel::LLMCompiledModel(const std::shared_ptr<ov::Model>& m
 
     m_cfg.update(any_copy(npuw_llm_props));
 
-    LOG_DEBUG("1. Creating kvcache model as clone of passed one.");
+    int step_msg = 0;
```

Comment:
I'd rather just remove the numbering, it is useless

Reply:
Fixed